MAINTAINERS.md

```diff
@@ -0,0 +1,38 @@
+# Core Maintainers of this repository
+
+| Name                  | Email                  | Kubernetes Slack         | GitHub                                            | Company           |
+|-----------------------|------------------------|--------------------------|---------------------------------------------------|-------------------|
+| Frederic Branczyk     | fbranczyk@gmail.com    | `@brancz`                | [@brancz](https://github.com/brancz)              | Red Hat           |
+| Lili Cosic            | coscilili@gmail.com    | `@lili`                  | [@lilic](https://github.com/lilic)                | Red Hat           |
+| Matthias Loibl        | mail@matthiasloibl.com | `@metalmatze`            | [@metalmatze](https://github.com/metalmatze)      | Red Hat           |
+| Paweł Krupa           | paulfantom@gmail.com   | ``                       | [@paulfantom](https://github.com/paulfantom)      | Red Hat           |
+| Paul Gier             | pgier@redhat.com       | ``                       | [@pgier](https://github.com/pgier)                | Red Hat           |
+| Sergiusz Urbaniak     | mail@matthiasloibl.com | ``                       | [@s-urbaniak](https://github.com/s-urbaniak)      | Red Hat           |
+| Simon Pasquier        |                        | ``                       | [@simonpasquier](https://github.com/simonpasquier)| Red Hat           |
+| Lucas Servén Marín    | lserven@gmail.com      | `@squat`                 | [@squat](https://github.com/squat)                | Red Hat           |
+| Vasily Sliouniaev     |                        | ``                       | [@vsliouniaev](https://github.com/vsliouniaev)    | Walmart Labs      |
+
+Please reach any of the maintainer on slack (#prometheus-operator on https://slack.k8s.io/) or email if you want to help.
+
+# Triage
+
+We also have some nice souls that help triaging issues and PRs. See [here](https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) for details about the role's permission.
+
+Full list of triage people is displayed below:
+
+| Name                  | Slack                    | GitHub                                                     | Company         |
+|-----------------------|--------------------------|------------------------------------------------------------|-----------------|
+| Noah Kantrowitz       | `@coderanger`            | [@coderanger](https://github.com/coderanger                | Geomagical/IKEA |
+
+
+## How to be maintainer?
+
+Any contributor that shows effort, consistentcy and willingness in maintaining this repository will be invited to join the Maintainers team.
+
+Open Source is all about the trust, which is the key factor in decision to add write permissions.
+
+Reach us if you have any questions on how to join maintainer team.
+
+## Initial authors
```

Comment:
I believe in the case of thanos maintainers file this section is about initial authors of the project.

Reply:
I see, I can change it to just reference this was copied or based on thanos file?

Reply:
Fixed.